pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -166,6 +172,11 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 			c.Logger.Errorf("Failed to update TaskRun status for PipelineRun %s: %v", pr.Name, err)
 			return err
 		}
+		pr.Status.SetCondition(&apis.Condition{
+			Type:   v1beta1.ConditionCleanedUp,
+			Status: corev1.ConditionTrue,
+			Reason: "Cleaned Up",
```

Comment:
The `Reasons` in other packages are typically CamelCased with no spaces.  See for example `pkg/pod/status.go` with `CouldntGetTask`, `TaskRunResolutionFailed`, `TaskRunValidationFailed`, etc...  Suggest moving this into a named constant and matching format to existing reasons.